app/Wiki.php

```diff
@@ -108,6 +113,11 @@ public function settings(): \Illuminate\Database\Eloquent\Relations\HasMany
         return $this->hasMany(WikiSetting::class);
     }
 
+    public function notification(): \Illuminate\Database\Eloquent\Relations\HasMany
+    {
+        return $this->hasMany(WikibaseNotificationSentRecord::class);
+    }
+
```

Comment:
This can be deleted as we already define the same relationship in `wikibaseNotificationSentRecord`